static/app/views/dashboards/widgetBuilder/buildSteps/groupByStep/groupBySelector.tsx

```diff
@@ -123,7 +123,7 @@ export function GroupBySelector({
           <DndContext
             collisionDetection={closestCenter}
             onDragStart={({active}) => {
-              setActiveId(active.id);
+              setActiveId(active.id.toString());
```

Comment:
This was the only change necessary, the type for ID changed from `string` to `string | number`